packages/poryscript/package.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: poryscript-pls
+description: |
+  Language server for poryscript (a high level scripting language for GBA-era Pokémon decompilation projects)
+homepage: https://github.com/huderlem/poryscript-pls
+licenses:
+  - MIT
```

Comment:
This project is currently unlicensed. So this  isn't correct. Part of including projects in Mason is they must have a proper open source license to guarantee we aren't breaking any sort of licensing rules. It might be worth reaching out to the developer to choose an open source license so other projects can properly support it.